src/actions/send-signup-confirmation-email.ts

```diff
@@ -1,40 +1,94 @@
 "use server";
 
+import type { SerializedEditorState } from "@payloadcms/richtext-lexical/lexical";
+import { convertLexicalToPlaintext } from "@payloadcms/richtext-lexical/plaintext";
 import { pretty, render, toPlainText } from "@react-email/render";
 import { addHours } from "date-fns";
 import { ShiftSignupConfirmationEmail } from "@/email/templates/ShiftSignupConfirmationEmail";
 import { createCalendarInvite } from "@/lib/email/create-calendar-invite";
 import { sendEmail } from "@/lib/email/send-email";
 
-export const sendSignupConfirmationEmail = async () => {
-  const start = addHours(new Date(), 2);
-  start.setMinutes(0);
-  const end = addHours(new Date(), 3);
-  end.setMinutes(0);
-
-  const invite = createCalendarInvite({
-    summary: "Tuesday Evening First Shift",
-    description: "You'll be doing stuff in this shift. Lets go team!",
-    start: start,
-    end: end,
-    location: "De Sering, Rhoneweg 6, 1043 AH Amsterdam",
-  });
+export type SendSignupConfirmationPayload = {
```

Comment:
Can you call this params or just anything that is not "payload"? because it might be confusing with the payload cms